helm/fiftyone-teams-app/templates/_rbac.tpl

```diff
@@ -0,0 +1,74 @@
+{{/*
+Create the name of the K8s Job Delegation Role
+*/}}
+{{- define "rbac-do-templates.role-name" }}
+{{- if .Values.apiSettings.rbac.role.name }}
+{{- .Values.apiSettings.rbac.role.name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := "do-management" }}
+{{- printf "%s-%s" (include "fiftyone-teams-app.fullname" .) $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create the name of the K8s Job Delegation Role Binding
+*/}}
+{{- define "rbac-do-templates.role-binding-name" }}
+{{- if .Values.apiSettings.rbac.roleBinding.name }}
+{{- .Values.apiSettings.rbac.roleBinding.name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := "do-management" }}
+{{- printf "%s-%s" (include "fiftyone-teams-app.fullname" .) $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create the name of the K8s Job Service Account
+*/}}
+{{- define "rbac-do-templates.service-account-name" }}
```

Comment:
Should we rename this, since the SA is no longer exclusively associated with the DO?